Handler/Sitemap.hs

```diff
@@ -11,10 +11,10 @@ getSitemapR :: Handler TypedContent
 getSitemapR = sitemap $ do
     priority 1.0 $ HomeR
 
-    priority 0.9 $ OldLtsR []
+    priority 0.9 $ OldStackageBranchR LtsBranch []
     -- TODO: uncomment when this is presentable
     --priority 0.9 $ DownloadR
```

Comment:
No, this should be removed

On Fri, Oct 16, 2015, 12:59 PM Konstantin Zudov <notifications@github.com>
wrote:

> In Handler/Sitemap.hs
> <https://github.com/fpco/stackage-server/pull/133#discussion_r42226661>:
>
> > @@ -11,10 +11,10 @@ getSitemapR :: Handler TypedContent
> >  getSitemapR = sitemap $ do
> >      priority 1.0 $ HomeR
> >
> > -    priority 0.9 $ OldLtsR []
> > +    priority 0.9 $ OldStackageBranchR LtsBranch []
> >      -- TODO: uncomment when this is presentable
> >      --priority 0.9 $ DownloadR
>
> @snoyberg <https://github.com/snoyberg> Is it still planned to make the
> download page 'presentable'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fpco/stackage-server/pull/133/files#r42226661>.
>